README.md

```diff
@@ -73,39 +75,80 @@ You will need to uncheck "Use Rosetta for x86_64/amd64 emulation on apple silico
 
 </div>
 
-This should start an agent server running on port 8080.
-
 <h3>Connect to Your Agent</h3>
 
-You can use the playground project to test with the server you just started.
+You can use the showcase voice agent, in `/playground` folder, to test with the server you just started.
 
-The Playground project is built on NextJS 14, hence it needs Node 18+.
+The project is built on NextJS 14, hence it needs Node 18 or later.
 
 ```bash
-# set up an .env file
+# Set up an .env file
 cp ./playground/.env.example ./playground/.env
 cd playground
 
-# install npm dependencies & start
+# Install npm dependencies & start
 npm i && npm run dev
 ```
 
-🎉 Congratulations! You now have our ASTRA powered voice agent running locally.
+🎉 Congratulations! You now have a ASTRA powered voice agent running locally.
 
-<br />
+<br>
 <h2>Agent Customization</h2>
 
-Our voice agent is a great place for you to start with, it uses following Extensions:
-
-- _agora_rtc_ / [Agora](https://docs.agora.io/en) for RTC transport + VAD + Azure speech-to-text (STT)
-- _azure_tts_ / [Azure](https://azure.microsoft.com/en-us/products/ai-services/text-to-speech) for text-to-speech (TTS)
-- _openai_chatgpt_ / [OpenAI](https://openai.com/index/openai-api/) for LLM
-- _chat_transcriber_ / A utility ext to forward chat logs into channel
-- _interrupt_detector_ / A utility ext to help interrupt agent
+To explore further, the ASTRA voice agent is an excellent starting point. It incorporates the following extensions, some of which will be interchangeable in the near future. Feel free to choose the ones that best suit your needs and maximize ASTRA’s capabilities.
 
 <div align="center">
 
-<image alt="ASTRA" src="./images/image-2.png">
+<table style="width: 100%;">
+  <tr>
+    <th align="center">Extension</th>
+    <!-- <th align="center">Interchangeable</th>  -->
+    <th align="center">Feature</th>
+    <th align="center">Description</th>
+  </tr>
+  <tr>
+  <td align="center">openai_chatgpt</td>
+    <!-- <td align="center">✅</td> -->
+    <td align="center">LLM</td>
+    <td align="center">Large Language Modal</td>
+  </tr>
+  <tr>
+  <td align="center">elevenlabs_tts</td>
+    <!-- <td align="center">✅</td> -->
+    <td align="center">Text-to-speech</td>
+    <td align="center">Convert text to audio</td>
+  </tr>
+  <tr>
+  <td align="center">azure_tts</td>
+    <!-- <td align="center">✅</td> -->
+    <td align="center">Text-to-speech</td>
+    <td align="center">Convert text to audio</td>
```

Comment:
same. maybe add vendor info otherwise the description column looks repetive.

Reply:
updated